data/json/npcs/refugee_center/NPC_Jenny_Forcette.json

```diff
@@ -13,7 +13,7 @@
     "worn_override": "REFUGEE_Jenny_worn",
     "carry_override": "REFUGEE_Jenny_carried",
     "weapon_override": "REFUGEE_Jenny_wield",
-    "traits": [ { "trait": "PACIFIST" }, { "trait": "Exp_Eng_Mechanical2" } ],
+    "traits": [ { "trait": "PACIFIST" }, { "trait": "Exp_Eng_Mechanical2" }, { "group": "Appearance_Caucasian" } ],
     "skills": [
```

Comment:
Jenny does not have common traits. Is is intended?

Reply:
Yep. Not really related to this PR though.